packages/openneuro-components/src/content/activity-slider-content.jsx

```diff
@@ -11,8 +11,7 @@ export const RecentData = {
             latestSnapshot: {
               tag: '1.0.0',
               description: {
-                Name:
-                  'Runabout: A mobile EEG study of auditory oddball processing in laboratory and real-world conditions',
+                Name: 'Runabout: A mobile EEG study of auditory oddball processing in laboratory and real-world conditions',
```

Comment:
I think this file may actually be mock content, as it's hard-coding most viewed and recently published, and these are definitely not what is shown there.

Reply:
Thanks, I've cleaned up the unused content. There's some only partially used content remaining but anything entirely unused is now removed.